job/storage/redis/redis.go

```diff
@@ -19,8 +19,14 @@ type DB struct {
 }
 
 // New instantiates a new DB.
-func New(address string, password redis.DialOption) *DB {
-	conn, err := redis.Dial("tcp", address, password)
+func New(address string, password redis.DialOption, passinterface int) *DB {
```

Comment:
ok, I update pull request using sendPassword bool instead of passinterface int. Hi, elimisteve, It looks good: )